client/src/components/ChangePassword.jsx

```diff
@@ -50,11 +51,14 @@ export class ChangePassword extends Component {
                     <Input
                       id="check_new_password"
                       value={ this.state.check_new_password}
+                      size={40}
                       onChange={ this.onChangeHandler }
                       placeholder="Confirm Password"
                       type="password"
                       label="confirm Password"
```

Comment:
I think "Confirm Password" or "Confirm password" seems prettier.

Reply:
gitlab have Confirm Password

Reply:
But you have 'confirm Password'